src/connector/src/source/google_pubsub/source/message.rs

```diff
@@ -0,0 +1,25 @@
+use bytes::Bytes;
+use google_cloud_pubsub::subscriber::ReceivedMessage;
+
+use crate::source::SourceMessage;
+
+impl From<ReceivedMessage> for SourceMessage {
+    fn from(message: ReceivedMessage) -> Self {
+        Self {
+            payload: {
+                let load = message.message.data;
+                match load.len() {
+                    0 => None,
+                    _ => Some(Bytes::from(load)),
+                }
+            },
+            // ! won't work yet -- ack_id is private
+            // reference: https://github.com/yoshidan/google-cloud-rust/issues/49
+            // ! offset: message.ack_id,
+            offset: 0.to_string(),
+
+            // ? what are the ramifications
+            split_id: 0.to_string(),
```

Comment:
we use split_id to identify a split.
Here its purpose is to note where the messages comes. We record the consumption progress in meta server in case we need to restore state in recovery and scale process.

Reply:
I see. Do you think a constant 0 `split_id` makes sense here then since any seeks will affect all readers anyways? 
In case we still want to label the messages with the virtual split readers on risingwave, we'll probably need to wrap `ReceivedMessage` to add that metadata.

Reply:
messages are now tagged with their origin split